tao_compiler/mlir/disc/transforms/conv_rewriter.cc

```diff
@@ -107,6 +107,72 @@ LogicalResult extractConvParams(mhlo::DynamicConvOp op, ConvParams& params) {
   return success();
 }
 
+struct QuantizedConvParams {
```

Comment:
better to re-use code from `ConvParams`, e.g. using a template struct?

Reply:
Have used template instead